src/core/os_glfw.h

```diff
@@ -204,12 +204,18 @@ bool lovrPlatformCreateWindow(const WindowFlags* flags) {
     return false;
   }
 
+
+#ifdef LOVR_LINUX_EGL
+  glfwWindowHint(GLFW_CONTEXT_CREATION_API, GLFW_EGL_CONTEXT_API);
+#endif
   glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 3);
   glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 3);
   glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
   glfwWindowHint(GLFW_OPENGL_FORWARD_COMPAT, GLFW_TRUE);
   glfwWindowHint(GLFW_OPENGL_DEBUG_CONTEXT, flags->debug);
+#ifndef LOVR_LINUX_EGL
   glfwWindowHint(GLFW_CONTEXT_NO_ERROR, !flags->debug);
```

Comment:
Curious, was this hint unavailable or otherwise not working with EGL?

Reply:
It failed to create the context if this was enabled.

```txt
Warning: Could not load module "graphics": EGL: Failed to create context: An unrecognized attribute or attribute value was passed in the attribute list
Error:
main.lua:8: attempt to index field 'graphics' (a nil value)
```